src/assets/scss/components/_starInput.scss

```diff
@@ -0,0 +1,63 @@
+.star-input {
+    &--active {
+        svg {
+            fill: $accent;
+        }
+        span {
+            color: $white;
+        }
+    }
+    &--highlighted {
+        transform: scale(1.25);
+    }
+    &__label {
+        position: relative;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: pointer;
+        &:focus {
+            svg {
+                fill: $accent;
+                transition: ease-in-out .5s;
+            }
+            span {
+                color: $white;
+                transition: ease-in-out .5s;
+            }
+        }
+    }
+    &__button {
+        // opacity: 0;
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: 1;
+        width: 100%;
+        height: 100%;
+        appearance: none;
+        opacity: 0;
+        &:checked ~ svg {
+            fill: $accent;
+        }
+        &:checked ~ span {
+            color: $white;
+        }
+    }
+    &__svg-star{
+        fill: $white;
+        height: 100px;
+        width: 100px;
+        margin: 10px;
+        padding: 10px;
+    }
+    &__star-number {
+        font-size: $font-size-small;
+        font-family: $h3-font;
+        font-weight: bold;
+        position: absolute;
+        top: 60px;
+        right: 65px;
```

Comment:
Rather than using magic numbers here you could do something like:

```
top: 0;
left: 0;
width: 100%;
height: 100%;
display: flex;
place-items: center;
```

Which would work no matter what the content height/width